### PR TITLE
CORDA-2157: Fix Windows build

### PR DESCRIPTION
**Pull request by vkolomeyko**
Or else tests:
AttachmentsClassLoaderTests.Test valid overlapping contract jar
ConstraintsPropagationTests.Happy path for Hash to Signature Constraint migration
are failing.

### CODE REVIEW

File: testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt [hunk 1]
```diff
@@ -89,13 +89,17 @@ object ContractJarTestUtils {
     }
 
     fun signContractJar(jarURL: URL, copyFirst: Boolean, keyStoreDir: Path? = null, alias: String = "testAlias", pwd: String  = "testPassword"): Pair<Path, PublicKey> {
+
```
Comment (josecoll) [inline testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt]: remove blank line

File: testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt [hunk 2]
```diff
@@ -89,13 +89,17 @@ object ContractJarTestUtils {
...
+        val urlAsPath = jarURL.toPath()
+
```
Comment (josecoll) [inline testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt]: ditto

File: testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt [hunk 3]
```diff
@@ -89,13 +89,17 @@ object ContractJarTestUtils {
...
         val jarName =
             if (copyFirst) {
-                val signedJarName = Paths.get(jarURL.path.substringBeforeLast(".") + "-SIGNED.jar")
-                Files.copy(jarURL.toPath(), signedJarName, StandardCopyOption.REPLACE_EXISTING)
+                val signedJarName = Paths.get(urlAsPath.toString().substringBeforeLast(".") + "-SIGNED.jar")
+
```
Comment (josecoll) [inline testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt]: ditto

File: testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt (hunk 1 above)
Comment (vkolomeyko) [reply testing/test-utils/src/main/kotlin/net/corda/testing/core/internal/ContractJarTestUtils.kt]: done

**Pull request merged**